static/app/components/sidebar/broadcasts.tsx

```diff
@@ -28,15 +28,77 @@ type Props = CommonSidebarProps & {
 
 type State = {
   broadcasts: Broadcast[];
-  error: boolean;
   loading: boolean;
 };
 
+function BroadcastSidebarContent({
+  orientation,
+  collapsed,
+  loading,
+  broadcasts,
+  whatIsNewRevampFeature,
+  hidePanel,
+  onResetCounter,
+}: {
+  broadcasts: Broadcast[];
+  loading: boolean;
+  onResetCounter: () => void;
+  whatIsNewRevampFeature: boolean;
+} & Pick<CommonSidebarProps, 'orientation' | 'collapsed' | 'hidePanel'>) {
+  useEffect(() => {
+    return () => {
+      if (whatIsNewRevampFeature) {
+        onResetCounter();
+      }
+    };
+  }, [onResetCounter, whatIsNewRevampFeature]);
```

Comment:
this is the only new code! basically if the user has access to the feature flag `organizations:what-is-new`, the `hasSeen` state will be reset on unmount. That is the change. The rest is the same